README.md

```diff
@@ -1 +1,34 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello Microverse
+
+> This is a simple project that doesn't have any complicated code.
+
+This project has two files, an HTML & a CSS file.
+
+## Built With
+
+- HTML & CSS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] Please host your project online (can be GITHUB PAGES) and update the live link demo to represent the project you have been working on. Hosting the project online helps those viewing it to interact with your project live.